numpy/core/shape_base.py

```diff
@@ -453,7 +449,7 @@ def stack(arrays, axis=0, out=None, *, dtype=None, casting="same_kind"):
     """
     if not overrides.ARRAY_FUNCTION_ENABLED:
         # raise warning if necessary
```

Comment:
Is the comment still needed?

Reply:
A, good catch.  Maybe it still makes sense, but changed it to `# reject non-sequences (and make tuple)`, since "warning" is definitely wrong now.